### PR TITLE
Tests refactoring and typo fixes

### PR DESCRIPTION
**Pull request by rudygodoy**
Minor typos fixes in comments and refactoring of tests to use `beforeEach` when possible.
Refs #259

### CODE REVIEW

File: test/StandardToken.js [hunk 1]
```diff
@@ -5,8 +5,13 @@ var StandardTokenMock = artifacts.require('./helpers/StandardTokenMock.sol');
 
 contract('StandardToken', function(accounts) {
 
+  let token;
```
Comment (rudygodoy) [inline test/StandardToken.js]: This refactoring might fail if the developer has previously ran tests that involve transactions within the testrpc environment. Because can ran out of funds before getting to complete all the test cases.
Not sure if the refactoring is worth it.

File: test/PullPayment.js [hunk 2]
```diff
@@ -1,15 +1,19 @@
 var PullPaymentMock = artifacts.require("./helpers/PullPaymentMock.sol");
 
 contract('PullPayment', function(accounts) {
-
+  let ppce;
+  let tAMOUNT = 17*1e18;
```
Comment (maraoz) [inline test/PullPayment.js]: why use the name tAMOUNT? I don't like it

File: test/StandardToken.js (hunk 1 above)
Comment (maraoz) [inline test/StandardToken.js]: I think that's OK. 100 wei is a very small value. why do you think it could run out of funds?

Comment (rudygodoy) [inline test/StandardToken.js]: It happened once to me because I was using the same 'testrpc' session for all tests, solved restarting it.

File: test/PullPayment.js (hunk 2 above)
Comment (rudygodoy) [inline test/PullPayment.js]: To differentiate of the local variable used in the second test case. Yes, the name can be changed to be more descriptive. `t` came from transaction.

Comment (frangio) [inline test/PullPayment.js]: I agree, I think we prefer `amount`, in small caps even. By the way, why did you rename the other varable to `ppce`?

**Comment by frangio** [comment]
Thanks @rudygodoy!
